Rawl_B502/genelach_dáil_cormaic_&_úa_labrada.trig

```diff
@@ -58,43 +58,43 @@
     a foaf:Person;
     irishRel:nomName "Luguid";
     rel:childOf <#Labraid>;
-    owl:sameAs <http://example.com/LL/dal_corpri_arad.trig#Lugaid>.
+    owl:sameAs <http://example.com/LL/de_genelach_dail_chormaic_&_h-úa_labrada.trig#Lugaid>.
 
 <#Cathbud>
     a foaf:Person;
     irishRel:nomName "Cathdub";
     rel:childOf <#Labraid>;
-    owl:sameAs <http://example.com/LL/dal_corpri_arad.trig#Cathub>.
+    owl:sameAs <http://example.com/LL/de_genelach_dail_chormaic_&_h-úa_labrada.trig#Cathub>.
 
 <#Cóeldub>
     a foaf:Person;
     irishRel:nomName "Cóeldub";
     rel:childOf <#Labraid>;
-    owl:sameAs <http://example.com/LL/dal_corpri_arad.trig#Caeldub>.
+    owl:sameAs <http://example.com/LL/de_genelach_dail_chormaic_&_h-úa_labrada.trig#Caeldub>.
 
 <#Cainnech>
     a foaf:Person;
     irishRel:nomName "Cainnech";
     rel:childOf <#Labraid>;
-    owl:sameAs <http://example.com/LL/dal_corpri_arad.trig#Cainnech>.
+    owl:sameAs <http://example.com/LL/de_genelach_dail_chormaic_&_h-úa_labrada.trig#Cainnech>.
 
 <#Cáirthenn>
     a foaf:Person;
     irishRel:nomName "Cáirthenn";
     rel:childOf <#Labraid>;
-    owl:sameAs <http://example.com/LL/dal_corpri_arad.trig#Cáirthend>.
+    owl:sameAs <http://example.com/LL/de_genelach_dail_chormaic_&_h-úa_labrada.trig#Cáirthend>.
 
 <#NioCuillenn>
     a foaf:Person;
     irishRel:nomName "Nio Cuillenn";
     rel:childOf <#Labraid>;
-    owl:sameAs <http://example.com/LL/dal_corpri_arad.trig#NioCulend>.
```

Comment:
There is also a link to what was formerly the Dál Corpri Arad here: http://example.com/Rawl_B502/item_úi_máelhuidir_2.trig#NiaCuilind